search-service/src/test/kotlin/com/egm/stellio/search/authorization/SubjectReferentialServiceTests.kt

```diff
@@ -278,21 +278,21 @@ class SubjectReferentialServiceTests : WithTimescaleContainer {
 
```

Comment:
I was also expecting at least one new test to check this transitive retrieving of global roles